OmniKit/Model/Pod.swift

```diff
@@ -57,13 +57,15 @@ public struct Pod {
     // Minimum duration of a single basal schedule entry
     public static let minimumBasalScheduleEntryDuration = TimeInterval.minutes(30)
 
-    // Amount of insulin delivered with 1 second between pulses for priming
-    public static let primeUnits = 2.6
+    // Default amount for priming bolus using secondsPerPrimePulse timing.
+    // Can potentially be overrided with value returned by pod during the pairing process.
+    public static var primeUnits = 2.6
```

Comment:
These should be constants. There should be no modifiable state here.

Reply:
Changed